Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -3240,6 +3269,20 @@ public static String getSdkVersionNumber() {
         return io.branch.referral.BuildConfig.VERSION_NAME;
     }
 
+    private final PurchasesUpdatedListener purchasesUpdatedListener = new PurchasesUpdatedListener() {
+        @Override
+        public void onPurchasesUpdated(BillingResult billingResult, @Nullable List<Purchase> purchases) {
+            if (billingResult.getResponseCode() == BillingClient.BillingResponseCode.OK && purchases != null) {
+                for (Purchase purchase : purchases) {
+                    if (prefHelper_.isAutoLogInAppPurchasesAsEventsEnabled()) {
```

Comment:
If it's false, do we assume customer calls this on their own?

If so, we should provide them a callback to let them know the state of their purchase status. (imo)